### ISSUE TITLE
Color codes captured in Set-Clipboard

### ISSUE THREAD

**Issue by FireInWinter** [PowerShell/PowerShell#17249]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

OS: Windows 10, Windows Server 2019
PowerShell Version: 7.2.2, 7.2.3

When outputting data to the clipboard, all color codes are captured as well.  This makes it really inconvenient to use the clipboard as the contents are now mangled.   These should be stripped when put in the clipboard as they have no use whatsoever as far as I can tell.  This is just a simple example of it happening as it happens even when formatting text to be sent in an email (first place I noticed it). 
Also Out-String shouldn't be needed either, but that's another ticket (#16431).

```
Get-PSDrive | Where-Object Name -ne 'Temp' | Select-Object Name, root | Out-String | Set-Clipboard
```
Then paste it back to notepad, pwsh, Word, or any other app (or at least I haven't found one it looks decent on).

### Expected behavior

```console
Name     Root
----     ----
Alias    
C        C:\
Cert     \
D        D:\
Env      
Function 
HKCU     HKEY_CURRENT_USER
HKLM     HKEY_LOCAL_MACHINE
Variable 
WSMan
```

### Actual behavior

```console
[32;1mName     Root[0m
[32;1m----     ----[0m
Alias    
C        C:\
Cert     \
D        D:\
Env      
Function 
HKCU     HKEY_CURRENT_USER
HKLM     HKEY_LOCAL_MACHINE
Variable 
WSMan
```

### Environment data

```powershell
Name                           Value
----                           -----
PSVersion                      7.2.3
PSEdition                      Core
GitCommitId                    7.2.3
OS                             Microsoft Windows 10.0.19044
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by jhoneill** [comment]
In 7.2.1 the default for `$PSStyle.OutputRendering`  was changed from `host`  (which had a reported theoretical security issue) to `ansi`. 
Ansi sequences are now added to all table headers - even if you set the style to no coloring in a reset code is still sent -  
 and the change of default means they are not removed when output is sent to a file, to the clipboard etc, resulting in "Ansi corrupted" output.  There are multiple issues about different forms of this .

As I understand it @SteveL-MSFT  made a change to fix the "security issue" which went into 7.2.2 but as of 7.2.3 the default has not reverted, so PowerShell is still not sending clean data. The work round is to set 
`$PSStyle.OutputRendering = "Host"`    
or    
`$PSStyle.OutputRendering = "PlainText"`

@SteveL-MSFT   is there a plan to change the default back to host ?

**Comment by daxian-dbw** [comment]
It's already changed back in a not-so-recent PR https://github.com/PowerShell/PowerShell/pull/16811, so resolve this issue as fixed.

**Comment by jhoneill** [comment]
@daxian-dbw  #16811 fixes the regex, but the default in  7.2.3 is ANSI , not Host
![image](https://user-images.githubusercontent.com/16209593/166916082-53efc392-d2fd-45a5-8058-42cb9e03cdb8.png)
